tests/functional/object/mcg/ui/test_mcg_ui.py

```diff
@@ -417,6 +417,7 @@ def test_obc_creation_and_deletion(
         assert test_obc.check_resource_existence(should_exist=False)
 
 
+@tier1
 @black_squad
 @tier1
 @pytest.mark.polarion_id("OCS-6334")
```

Comment:
are all of these tests under one test case Polarion id? not sure analysis will be traceable to specific test in spreadsheet and RP. please check